Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.cxx

```diff
@@ -1166,11 +1166,10 @@ void qMRMLSegmentEditorWidget::updateWidgetFromMRML()
 
   if (!d->ParameterSetNode)
     {
-    qCritical() << Q_FUNC_INFO << ": Invalid segment editor parameter set node";
```

Comment:
@lassoan This is to fix the from the steps in https://mantisarchive.slicer.org/view.php?id=4664.
The other solution would be to check if the parameter node exists before calling updateWidgetFromMRML from setMRMLScene.

Reply:
I agree that it makes sense to remove the error message here but before returning we should probably disable the widgets that we would normally update with valid content in this method.